test/jdk/javax/swing/JSpinner/TestJSpinnerPressUnpress.java

```diff
@@ -23,7 +23,7 @@
 
 /*
  * @test
- * @bug 8234733
+ * @bug 8234733 8260291
```

Comment:
There is no need to add bug id 8260291 here.
http://openjdk.java.net/jtreg/faq.html#when-should-i-update-the-bug-entry-in-a-test-description

Reply:
ok, makes sense. I have removed the bugID.